src/wp-includes/pomo/plural-forms.php

```diff
@@ -239,13 +239,13 @@ protected function parse( $str ) {
 		 * @param int $num Number to get plural form for.
 		 * @return int Plural form value.
 		 */
-		public function get( $num ) {
-			if ( isset( $this->cache[ $num ] ) ) {
-				return $this->cache[ $num ];
-			}
-			$this->cache[ $num ] = $this->execute( $num );
-			return $this->cache[ $num ];
-		}
+                public function get( $num ) {
+                        if ( isset( $this->cache[ (int) $num ] ) ) {
+                                return $this->cache[ $num ];
```

Comment:
This this one also be cast to `(int)`?

If so, can we add `$num = (int) $num` at the start of the `get()` method instead of repeating the `(int)` cast?

Reply:
We can. Good idea!